src/main/java/ResumeGame.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
+public class ResumeGame {
+  private static final String REGAME_PHRASE = "게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요. ";
+  private static final int ENDING_USERINPUT = 2;
+
+  private boolean isResume = true;
+
+  public boolean askResume() {
+    System.out.print(REGAME_PHRASE);
+    Scanner scanner = new Scanner(System.in);
+    int isResume = Integer.parseInt(scanner.nextLine());
+    if (isResume == ENDING_USERINPUT) {
+      return false;
+    } else {
+      return true;
+    }
```

Comment:
```suggestion
    return isResume != ENDING_USERINPUT
```
이런 이유로 `else` 구문을 쓸 필요가 없다는 뜻입니다.